Assets/Scripts/Puzzles/Color/ColorPuzzleSettings.cs

```diff
@@ -0,0 +1,30 @@
+﻿using UnityEngine;
+
+namespace DN.Puzzle.Color
+{
+	public class ColorPuzzleSettings : ScriptableObject
+	{
+		[System.Serializable]
+		public struct ColorLineSettings
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Implement 'IEquatable<T>' in value type 'ColorLineSettings'.](https://app.codacy.com/manual/stepperman/pvb-programming/pullRequest?prid=5403921)